src/internal/utils/exec.go

```diff
@@ -17,10 +18,10 @@ const colorCyan = "\x1b[36;1m"
 const colorWhite = "\x1b[37;1m"
 
 //nolint
-func ExecCommand(showLogs bool, envVariables []string, commandName string, args ...string) (string, error) {
+func ExecCommandWithContext(ctx context.Context, showLogs bool, commandName string, args ...string) (string, error) {
```

Comment:
I know it wasn't added by this PR but am I correct in understanding this `showLogs` toggle is copying logs directly  to stdout/stderr?